pkg/virt-api/webhooks/validating-webhook/admitters/vmclone-admitter.go

```diff
@@ -108,6 +110,10 @@ func (admitter *VirtualMachineCloneAdmitter) Admit(ar *admissionv1.AdmissionRevi
 		causes = append(causes, newCauses...)
 	}
 
+	if newCauses := validateCloneHostNameNotConformingToDNSLabelRules(vmClone); newCauses != nil {
```

Comment:
In the future we may add more general validations to hostname
```suggestion
	if newCauses := validateCloneHostName(vmClone); newCauses != nil {
```

Reply:
done